### PR TITLE
Close #899: Update styles for the members list, remove kebab menus

### LINKED ISSUE #899: Update styles for the members list, remove kebab menus
### Problem

Now that we have a profile popover (see issue #798), there's no need to show a kebab menu next to members names (on the Dashboard sidebar) or to show buttons with actions on the "All members" page. Here's what I'm talking about:
Captura de ecrã 2020-04-14, às 10 44 47
Captura de ecrã 2020-04-14, às 10 45 11

### Solution
Components changed:
- [Dashboard sidebar](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=48%3A70);
- [Group members list (desktop)](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=1715%3A92);

Captura de ecrã 2020-04-19, às 18 20 23
Captura de ecrã 2020-04-19, às 18 21 23

### PR DESCRIPTION
**Pull request by pieer**

### CODE REVIEW

File: strings/french.strings [hunk 1]
```diff
@@ -84,9 +84,6 @@
 /* MISSING TRANSLATION - views/containers/contributions/GroupPledgesGraph.vue */
 "{span_}Group{_span} goal" = "{span_}Group{_span} goal";
 
-/* MISSING TRANSLATION - views/containers/dashboard/GroupMembersAllModal.vue, views/containers/dashboard/GroupMembersMenu.vue, views/pages/DesignSystem.vue */
-"Remove member" = "Remove member";
-
```
Comment (taoeffect) [inline strings/french.strings:89]: Hmm, I'm not sure why these `.strings` files have been modified... could it be an outdated commit? Can you try merging `master` and see if that fixes this?

Comment (pieer) [reply strings/french.strings:89]: The file don't exist anymore

**Review by taoeffect — DISMISSED** [review]
Nice job @pieer 👍

File: test/cypress/integration/group-member-removal.spec.js [hunk 2]
```diff
@@ -17,8 +17,11 @@ describe('Group - Removing a member', () => {
     cy.getByDT('groupMembers').find(`ul>li:nth-child(${nth})`).within(() => {
       cy.getByDT('username').should('contain', `${username}-${userId}`)
 
-      cy.getByDT('menuActions').find('button').click()
-      cy.getByDT('menuContent').find('ul>li:nth-child(2) button')
+      cy.getByDT('openMemberProfileCard').click()
+    })
+
+    cy.getByDT('memberProfileCard').within(() => {
+      cy.getByDT('buttonRemoveMember')
```
Comment (sandrina-p) [inline test/cypress/integration/group-member-removal.spec.js:24]: A quick tip when using `data-test`. You can write just `removeMember` and use `button` as the second parameter to make sure it's indeed a `<button>`. `cy.getByDT('removeMember', 'button')`

**Review by sandrina-p — DISMISSED** [review]
Looks good to me!

File: strings/french.strings (hunk 1 above)
Comment (sandrina-p) [reply strings/french.strings:89]: I don't understand what causes these changes. It happens the same too me and I never touched the files...

@taoeffect maybe you should create a PR to explicitly say how to update the strings and it should be a quick and straightforward way without the need to manually install other stuff.

**Review by mmbotelho — CHANGES_REQUESTED** [review]
The "remove kebab menus" part is done! 

However, it's missing the "update styles for the member's  list" 😄 

Captura de ecrã 2020-05-26, às 14 32 17

Captura de ecrã 2020-05-26, às 14 32 44

Captura de ecrã 2020-05-26, às 14 33 48

Captura de ecrã 2020-05-26, às 14 33 33

**Review by mmbotelho — DISMISSED** [review]

**Review by taoeffect — APPROVED** [review]
Great job @pieer!

**Pull request merged**